lib/runtime/test_helpers.go

```diff
@@ -270,6 +271,21 @@ func (trs *TestRuntimeStorage) GetChild(keyToChild []byte) (*trie.Trie, error) {
 // ClearPrefix ...
 func (trs *TestRuntimeStorage) ClearPrefix(prefix []byte) {
 	trs.trie.ClearPrefix(prefix)
+
+	iter := trs.db.NewIterator()
+
+	for iter.Next() {
```

Comment:
I think you can seek to a particular key or prefix. https://godoc.org/github.com/dgraph-io/badger#Iterator.Seek

Reply:
good point, I'll open an issue for this on chainDB to add it to the iterator interface